src/acuvity/apexextend.py

```diff
@@ -35,7 +55,14 @@ def list_analyzer_names(self, group: Optional[str] = None) -> List[str]:
         """
         if self._available_analyzers is None:
             self._available_analyzers = self.list_analyzers()
-        return sorted([ a.id for a in self._available_analyzers if (group is None or a.group == group) and a.id is not None ])
+        analyzer_ids = [a.id for a in self._available_analyzers
+                       if (group is None or a.group == group) and a.id is not None]
+
+        # Map IDs to friendly names and filter out any that aren't in our mapping
+        mapped_names = [self.analyzer_id_name_map.get(id_) for id_ in analyzer_ids]
+
+        # Filter out None values and sort
+        return sorted([name for name in mapped_names if name is not None])
```

Comment:
not sure I understand why you're changing this. It looks like you're looking for an additional list that lists the friendly names? Then this should be a separate function. The purpose of this function is to have a list of names that can be passed to analyzers during scan.